src/main/osd/osd.c

```diff
@@ -75,6 +75,7 @@
 #include "flight/imu.h"
 #include "flight/mixer.h"
 #include "flight/position.h"
+#include "flight/position_control.h"
```

Comment:
What is this for? Is osd.c dependent on something in position_control.h?

Reply:
Removed